dev-libs/isl/isl-0.19.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Library for manipulating integer points bounded by affine constraints"
+DESCRIPTION="isl is a thread-safe C library for manipulating sets and \
+relations of integer points bounded by affine constraints.  The descriptions \
+of the sets and relations may involve both parameters and existentially \
+quantified variables.  All computations are performed in exact integer \
+arithmetic using GMP."
+HOMEPAGE="http://isl.gforge.inria.fr"
+COPYRIGHT="2015-2016 Polly Labs
+	2014-2015 INRIA Rocquencourt
+	2012-2014 Ecole Normale Superieure
+	2011 Sven Verdoolaege
+	2010-2011 INRIA Saclay
+	2008-2009 Katholieke Universiteit Leuven
+	2006-2007, 2012 Universiteit Leiden"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="http://isl.gforge.inria.fr/isl-$portVersion.tar.bz2"
+CHECKSUM_SHA256="d59726f34f7852a081fbd3defd1ab2136f174110fc2e0c8d10bb122173fa9ed8"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=19.0.0
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	isl$secondaryArchSuffix = $portVersion
+	lib:libisl$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	isl${secondaryArchSuffix}_devel = $portVersion
+	devel:libisl$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	isl$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_devel="
+	isl13${secondaryArchSuffix}_devel
```

Comment:
@Begasus, I don't know if there is a strict rule for this, but my understanding is that we prefer to put a `CONFLICTS isl` in isl13's recipe, instead of a `CONFLICTS isl13` in isl's recipe.
(That's what I have learned from @korli.)
So, I think we should remove the `CONFLICTS` in `dev-libs/isl/isl-0.19.recipe` and, instead, add this one in `isl13-0.14.recipe`:
```
CONFLICTS_devel="
	isl${secondaryArchSuffix}_devel
	"
```
What's nice about this is that a single `CONFLICTS` can handle everything, if put in the right recipe.

Reply:
I already thought it looked strange in here, will remove it when I get around to it :)

Reply:
Why do we even have isl13 (for version 0.14)?

Reply:
Ok, but is anything relying on soname 13 for isl? The only thing I see using isl directly in the recipe is cloog (which builds fine with latest isl version) ...

Reply:
Good catch.
In that case we can probably remove `dev-libs/isl/isl13-0.14.recipe`.
(I guess anyone building anything outside haikuports and linking against `libisl.so.13` provided by `isl13-0.14.recipe` would be able to build against any newer libisl.)

Reply:
I'll keep it in for now, until someone comes up with a reason to keep it in :)